### PR TITLE
update sorting | responsiveness fixes

### PR DESCRIPTION
**Pull request by waqas-mumtaz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,10 +18,11 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.gitignore
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
+yarn.lock
```
Comment (lsc0498) [inline .gitignore:26]: We better don't "gitignore" `yarn.lock` and `.gitignore` 😉

File: src/components/course/SortParticipant.tsx [hunk 2]
```diff
@@ -1,72 +1,122 @@
...
+                  />
+                </AntdButton>
+              </li>
+            </li>
+          </ul>
+        </div>
+      </>
+    );
+  }
+);
```
Comment (lsc0498) [inline src/components/course/SortParticipant.tsx:121]: There are again many linter errors in this file. Please make sure you have merged `master` into this branch.

File: yarn.lock [hunk 3]
```diff
@@ -2,34 +2,34 @@
 # yarn lockfile v1
```
Comment (lsc0498) [inline yarn.lock:3]: Was updating `yarn.lock`on purpose?

File: src/components/course/SortParticipant.tsx [hunk 4]
```diff
@@ -1,72 +1,122 @@
...
+              <AntdButton onClick={() => sort('email')}>
+                E-Mail-Addresse
+                <Icons.SortSolid
+                  style={{ marginLeft: '5px', height: '12px' }}
+                />
+              </AntdButton>
+            </li>
+            <li>
+              <AntdButton onClick={() => sort('grade')}>
+                Note
```
Comment (lsc0498) [inline src/components/course/SortParticipant.tsx:101]: Here "grade" should be translated with "Klasse" instead of "Note".

**Review by lsc0498 — CHANGES_REQUESTED** [review]
There is still a slight problem at some screen widths:
Bildschirmfoto 2021-03-14 um 03 35 18
The overall design looks nice though 👍🎉

**Comment by lsc0498** [comment]
And please make sure to resolve merge conflicts 😉

**Comment by waqas-mumtaz** [comment]
> There is still a slight problem at some screen widths:
> Bildschirmfoto 2021-03-14 um 03 35 18
> The overall design looks nice though 👍🎉

@lsc0498 can you please mention the screen size?

**Comment by lsc0498** [comment]
> > There is still a slight problem at some screen widths:
> 
> > Bildschirmfoto 2021-03-14 um 03 35 18
> 
> > The overall design looks nice though 👍🎉
> 
> 
> 
> @lsc0498 can you please mention the screen size?

Just try different screen widths in the responsive mode of your browser. E.g. in Google Chrome you can change the width continuously in the development mode.

**Review by lsc0498 — APPROVED** [review]
Looks good now. Thank you!🙂

**Pull request merged**